### ISSUE TITLE
Game Crash

### ISSUE THREAD

**Issue by alecfifteen** [hlysine/create_connected#82]
Hello so i am making a minecraft server for me and some friends and i added the Create : connected mod, but it keeps on crashing our Server and our actual games, even when we go into singleplayer. it's both a multiplayer and singleplayer issue. 

and here is the Logs.

[latest.log](https://github.com/user-attachments/files/17138929/latest.log)

**Comment by hlysine** [comment]
I don't see a crash report in the logs, and the errors shown in the logs are unrelated to Create: Connected. Are you sure you only get the crash when Create: Connected is installed?

**Comment by alecfifteen** [comment]
Yes i am sure, when ever i remove it, it allows me to join singleplayer and start up the server and im just here at a chokepoint not knowing what to do

**Comment by hlysine** [comment]
Stale issue, closing. If you are still having this crash, please open a new issue with crash reports attached.
